pkg/skaffold/test/structure/get_image.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package structure
+
+import (
+	"context"
+	"io"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/build"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/docker"
+)
+
+var GetImagefn = GetImage
+var ResolveArtifactImageTagfn = resolveArtifactImageTag
+
+// GetImage downloads the image for container-structure-test
+func GetImage(ctx context.Context, out io.Writer, imageName string, bRes []build.Artifact, localDaemon docker.LocalDaemon,
```

Comment:
consider naming this as "getImage" to make it private.

Reply:
if its used in tests outside of this package, it would be good to look into moving the test inside this package.

Reply:
Made `getImage()` private.